lib/Minilla/Release/BumpVersion.pm

```diff
@@ -76,6 +88,25 @@ sub default_new_version {
     }
 }
 
+sub check_version_compatibility {
+    my ($curr, $next) = @_;
+
+    return version_format($curr) eq version_format($next)
```

Comment:
I'm not sure, is it possible both `$curr` and `$next` are `"unknown"` case ?

Reply:
I don't think the `version::is_lax` called in the previous process allows the original version format, but anyway, if someone is using their own version format, we don't need to check it, so this is how it is handled.

Reply:
I see